### DISCUSSION TITLE
Very perfect course

### DISCUSSION THREAD

**Discussion by ababyprogrammer** [Cyfrin/security-and-auditing-full-course-s23#87] [General]
Hello there!👋🏻

It was very useful for me in programming. 👨🏻‍💻

**Thanks.**  🙏🏻

**Comment by cromewar** [comment]
Big shoutout to @PatrickAlphaC for making this a reality. All the credit to him!

**Reply by PatrickAlphaC** [reply]
And the help of the Cyfrin team :)
